From c8822aa772f07ee5d68b6648278a5622b4d2f3b6 Mon Sep 17 00:00:00 2001
From: Mahesh Mahadevan <r9aadq@freescale.com>
Date: Mon, 7 Apr 2008 16:02:17 -0500
Subject: [PATCH] ENGR00071731 Fix MX37 DDR Clock Get Function

The fixes the DDR clock value returned by the clocking API

Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>
---
 arch/arm/mach-mx37/clock.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx37/clock.c b/arch/arm/mach-mx37/clock.c
index 569eb1b..d970355 100644
--- a/arch/arm/mach-mx37/clock.c
+++ b/arch/arm/mach-mx37/clock.c
@@ -1890,7 +1890,7 @@ static void clk_tree_init(void)
 	} else if (reg == 1) {
 		ddr_clk.parent = &axi_b_clk;
 	} else if (reg == 2) {
-		ddr_clk.parent = &axi_b_clk;
+		ddr_clk.parent = &axi_c_clk;
 	} else {
 		ddr_clk.parent = &emi_core_clk;
 	}
-- 
1.5.4.4

